src/main/resources/languages/english.xml

```diff
@@ -442,10 +442,15 @@
 	</string>
 
 	<string>
-		<key>FileTypeImage</key>
+		<key>FileChooser.FileTypeImage</key>
 		<value>Image (jpg/bmp/png/gif)</value>
 		<hint>file open dialog</hint>
 	</string>
+	<string>
```

Comment:
If possible when renaming a `<key>` (if the `<value>` is not modified then as the translations remain valid, and if already done) also rename the `<key>` in other languages. xml who have it.

But if the `<value>` changes then the translation will surely no longer be suitable and therefore there possibly add a `<!-- translation value to review-->` in the other .xml files of the other languages ...

Reply:
Good point, done !

Reply:
if the value changes it could simply be a more accurate translation.  Probably there is a way to prepare a release candidate and then diff the language file with the last official release to see which translations have changed.